module-infrastructure/security/src/main/kotlin/site/yourevents/filter/CustomExceptionHandleFilter.kt

```diff
@@ -0,0 +1,44 @@
+package site.yourevents.filter
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import jakarta.servlet.FilterChain
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.jetbrains.annotations.NotNull
+import org.springframework.web.filter.OncePerRequestFilter
+import site.yourevents.error.exception.ServiceException
+import site.yourevents.response.ApiResponse
+import site.yourevents.type.ErrorCode
+
+class CustomExceptionHandleFilter : OncePerRequestFilter() {
+    override fun doFilterInternal(
+        @NotNull request: HttpServletRequest,
+        @NotNull response: HttpServletResponse,
+        @NotNull filterChain: FilterChain
```

Comment:
해당 부분은 [저번 PR](https://github.com/Leets-Official/Yes-BE/pull/10#discussion_r1903608530)의`@Value`의 컨벤션과는 다르게 `@NotNull`은 위와 같이 적용되나요?

Reply:
저번에 말씀드린 `@Value` 컨벤션은 생성자 주입 시 환경변수 경로에 따라 가독성을 해칠 수 있어서 개행을 적용하자고 건의드렸습니다..!! 
그리고  `@NotNull`의 경우, 오히려 개행을 적용하면 가독성이 저하될 것 같아 해당 방식으로 적용하였는데, 어떻게 생각하시나요? 
```suggestion
        @NotNull 
        request: HttpServletRequest,
    
        @NotNull 
        response: HttpServletResponse,
        
        @NotNull 
        filterChain: FilterChain
```
사실 해당 부분에 대해 충분한 논의가 이루어지지 않아 계속 코멘트가 달리는 것 같습니다..ㅠㅠ 조속한 시일 내에 명확한 컨벤션을 정해야 할 듯 싶습니다.. 😭

Reply:
저도 같은 생각입니다. 하지만, 이 부분은 통일을 하여 혼동을 줄일 필요가 있다고 생각합니다!
가독성을 고려하면, 어노테이션만 붙는 경우는 개행을 추가하지 않고, 어노테이션 뒤에 부수적인 값이 붙으면 개행을 하는 방법이 좋다고 생각하는데, 어떻게 생각하시나요?
코드로 정리하면 다음과 같습니다.
```kotlin
@NotNull request: HttpServletRequest,
@NotNull response: HttpServletResponse,
@NotNull filterChain: FilterChain
```

```kotlin
@Value("\${sentry.dsn}") 
private val sentryDsn: String,

@Value("\${sentry.environment}") 
private val environment: String,

@Value("\${sentry.servername}") 
private val serverName: String
```

Reply:
제가 생각하던 이상적인 어노테이션 컨벤션입니다..🥹

지금 예찬님이 제시해주신 방식 그대로 적용하면 좋을 듯 싶습니다!!